### PR TITLE
Improve italian form for /partners

### PR DESCRIPTION
**Pull request by Fale**
As title

### CODE REVIEW

File: content/it/partners/_index.html [hunk 1]
```diff
@@ -1,135 +1,43 @@
...
-class: gridPage
-cid: partners
----
-<meta charset="utf-8" />
-
-<section id="users">
-    <main>
-        <h5>Kubernetes lavora in collaborazione con diversi partner al fine di creare un codice dalla base forte e dinamica che supporti uno spettro di piattaforme complementari.</h5>
+    <main class="main-section">
+        <h5>Kubernetes collabora con i partner per creare un sorgente forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
```
Comment (fabriziopandini) [inline content/it/partners/_index.html:11]: What about "..per creare un codebase che supporti uno spettro di piattaforme complementari."

File: content/it/partners/_index.html [hunk 2]
```diff
@@ -1,135 +1,43 @@
...
         <div class="col-container">
           <div class="col-nav">
             <center>
               <h5>
-                <b>Fornitori di Servizi Certificati Kubernetes</b>
+                <b>Fornitori Certificati di Servizi su Kubernetes</b>
```
Comment (fabriziopandini) [inline content/it/partners/_index.html:16]: What about: "Fornitori di servizi Certificati su Kubernetes"

File: content/it/partners/_index.html [hunk 3]
```diff
@@ -1,135 +1,43 @@
...
               </h5>
-              <br>Fornitori di servizi controllati con grande esperienza nell'aiutare le imprese ad adottare con successo Kubernetes.
+              <br>Fornitori verificati di servizi con grande esperienza nell'aiutare le imprese ad adottare con successo Kubernetes.
```
Comment (fabriziopandini) [inline content/it/partners/_index.html:18]: remove "verificati" (it is already in the title and in Italian, the phrase sounds weird)

File: content/it/partners/_index.html [hunk 4]
```diff
@@ -1,135 +1,43 @@
...
+              <button id="kcsp" class="button" onClick="updateSrc(this.id)">Guarda i Partners KCSP</button>
+              <br><br>Interessato a diventare un partner <a href="https://www.cncf.io/certification/kcsp/">KCSP</a>?
             </center>
           </div>
           <div class="col-nav">
             <center>
               <h5>
-                <b>Distribuzioni Kubernetes Certificate, Piattaforme Hosted e Installers</b>
-              </h5>La conformità del software assicura che la versione di Kubernetes di ogni fornitore supporti le API richieste.
+                <b>Distribuzioni Kubernetes Certificate, Piattaforme Hosted e Software di installazione</b>
```
Comment (fabriziopandini) [inline content/it/partners/_index.html:27]: what about "Distribuzioni di Kubernetes Certificate, Certified Hosted Platforms and Software di installazione Certificati"

File: content/it/partners/_index.html [hunk 5]
```diff
@@ -1,135 +1,43 @@
...
+              </h5>La conformità del software assicura che la versione di Kubernetes di ogni fornitore supporti le API necessarie.
               <br><br><br>
               <button id="conformance" class="button" onClick="updateSrc(this.id)">Guarda i Partner di Conformità</button>
```
Comment (fabriziopandini) [inline content/it/partners/_index.html:30]: "Guarda i partner certificati"

File: content/it/partners/_index.html [hunk 6]
```diff
@@ -1,135 +1,43 @@
...
-              <button id="conformance" class="button" onClick="updateSrc(this.id)">vedi le conformita' Partners</button>
-              <br><br>Interessato a diventare un <a href="https://www.cncf.io/certification/software-conformance/">Certificato Kubernetes</a>?
-            </center>
-          </div>
-          <div class="col-nav">
-            <center>
-          <h5><b>Partner di formazione di Kubernetes</b></h5>
-              <br>Fornitori di formazione con esperienza che hanno una profonda esperienza nella formazione della tecnologia cloud nativa.
+              <h5><b>Partner per la Formazione su Kubernetes</b></h5>
+              <br>Professionisti esperti e certificati, con solida esperienza nella formazione su tecnologica Cloud Native.
```
Comment (fabriziopandini) [inline content/it/partners/_index.html:37]: nit tecnologica --> tecnologie

File: content/it/partners/_index.html [hunk 7]
```diff
@@ -1,135 +1,43 @@
...
               <br><br><br><br>
-              <button id="ktp" class="button" onClick="updateSrc(this.id)">vedi KTP Partners</button>
-              <br><br>Interessato a diventare un  <a href="https://www.cncf.io/certification/training/">KTP</a>?
+              <button id="ktp" class="button" onClick="updateSrc(this.id)">Guarda i Partner di KTP</button>
```
Comment (fabriziopandini) [inline content/it/partners/_index.html:39]: Guarda i KTP partner

File: content/it/partners/_index.html (hunk 2 above)
Comment (Fale) [reply content/it/partners/_index.html:16]: are you sure about this? I think that "Certified Service Providers" means that the provider is certified, not the service (that usually is consulting, so very difficult to certify)

Comment (fabriziopandini) [reply content/it/partners/_index.html:16]: "di servizi" in the middle between Certified and Kubernetes feels weird to me.
But we ca iterate later in the final review

**Pull request merged**